systest/backup/nfs-backup/docker-utils/dgraph-nfs-client/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:20.04
```

Comment:
We should avoid adding new docker files. Consider following this article instead https://blog.ruanbekker.com/blog/2020/09/20/setup-a-nfs-server-with-docker/

Reply:
done @aman-bansal